#! /bin/sh /usr/share/dpatch/dpatch-run
## 23_lintian-overrides.dpatch by Balint Reczey <balint@balintreczey.hu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: lintian override files

@DPATCH@
--- /dev/null	2009-07-20 11:31:15.504680204 +0200
+++ ./tshark.lintian-overrides	2009-07-20 14:30:50.000000000 +0200
@@ -0,0 +1,2 @@
+tshark binary: binary-or-shlib-defines-rpath
+
--- /dev/null	2009-07-20 11:31:15.504680204 +0200
+++ ./wireshark-common.lintian-overrides	2009-07-20 14:30:59.000000000 +0200
@@ -0,0 +1,2 @@
+wireshark-common binary: binary-or-shlib-defines-rpath
+
